src/config/NetworkConfig.ts

```diff
@@ -74,7 +74,7 @@ export const defaultTestnetNetworkConfig: NetworkConfig = {
         defaultDynamicFeeMultiplier: 100,
         epochAdjustment: 1637848847,
         totalChainImportance: 7842928625000000,
-        generationHash: '7FCCD304802016BEBBCD342A332F91FF1F3BB5E902988B352697BE245F48E836',
+        generationHash: 'A7D2BA9249F3F3874DC93E5790C4E4D4C934B0EDC3DE02D9DC593486688E75F9',
```

Comment:
It should be 49D6E1CE276A85B70EAFE52349AACCA389302E7A9754BCF1221E79494FC665A4

Reply:
`currencyMosaicId` and `harvestingMosaicId` should be 72C0212E67A08BCE

Reply:
ok, updated